packages/experimental/Button/src/Button.types.ts

```diff
@@ -114,6 +98,9 @@ export interface ButtonProps extends Omit<IWithPressableOptions<ViewProps>, 'onP
 
   /** A floating action button  */
   fab?: boolean;
+
+  /** Sets style of button to a preset size style  */
+  size?: ButtonSize
```

Comment:
ah the size thing here was from design, so it seems like, at least on web/win32, there's a sizing token to (some?) components. I can follow up with design if this will be a consistent thing.

Reply:
Sunmin said that if components have a "size," it'll be standardized to small/medium/large. So it'll be according to design/redlines, but we should be standardizing as we move to v1.

Reply:
So I guess... we can close the issue

Reply:
Does that mean that there will be a separate "size" global or control token that will eventually show up?
Or is that just... tbd? I was mostly curious if the idea from design is that "size" will be a theme level thing. But yeah closing the issue is fine, thanks for following up!

Reply:
It would be considered a control token, I don't think it would be a global. The size is basically an option for particular clients to override the default size if they want for particular components. For example win32 might want to use a medium button in certain places.